.gitignore

```diff
@@ -1,4 +1,3 @@
 .nextflow*
 work/
 data/
-.DS_Store
```

Comment:
I copied this file from the NGI-ChIPseq pipeline. Maybe we need to add it there also?